Simplenote.xcodeproj/project.pbxproj

```diff
@@ -1495,7 +1495,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				COPY_PHASE_STRIP = NO;
-				DEVELOPMENT_TEAM = PZYM8XX95Q;
+				DEVELOPMENT_TEAM = 79Z92ZJAS3;
```

Comment:
Can you please revert all changes to this project file? These shouldn't be merged.